haskell/snapshot.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+name: segmented-cfmm-snapshot
+
+resolver: lts-17.3
+
+packages:
```

Comment:
A lot of dependencies below. Likely we'll soon need them all, but at the moment I don't think it's the case (e.g. `morley-metadata` seem unnecessary right now).

Even tho we may use them later I think we should have this list as small as possible from the beginning.

Reply:
Removed all unneeded dependencies.